src/main/java/Duke.java

```diff
@@ -1,10 +1,54 @@
+import javax.sound.midi.SysexMessage;
+import java.util.Scanner;
+
 public class Duke {
+
+    static Task[] tasks = new Task[100];
+    static int taskPosition = 0;
+
+
+    public static void storeTask(Task t){
+        tasks[taskPosition] = t;
+        taskPosition++;
+        System.out.println("added: " + t.description);
+    }
+
+    public static void markAsDone(int taskIndex){
+        System.out.println("Nice! I've marked this task as done: ");
+        tasks[taskIndex-1].isDone = true;
+        System.out.println("[" + tasks[taskIndex-1].getStatusIcon()+ "]" + tasks[taskIndex-1].description);
+    }
+
+    public static void listArray(Task[] tasks){
+        int textNumber = 1;
+        for(Task t:tasks){
+            if(t != null){
+                System.out.println(textNumber + ".[" + t.getStatusIcon() + "] " + t.description);
+                textNumber++;
+            }
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm 343 Guilty Spark! You may call me Spark!");
+        System.out.println("What can I do for you?");
+        String text = "hi";
+        while(!text.equalsIgnoreCase("Bye")){
+            Scanner in = new Scanner(System.in);
+            text = in.nextLine();
+            Task t = new Task(text);
+            if(text.equalsIgnoreCase("List")){
+                listArray(tasks);
+            }
+            else if(text.startsWith("Done") || text.startsWith("done")){
+                Integer taskIndex = Integer.parseInt(text.substring(5));
+                markAsDone(taskIndex);
+
+            }
+            else{
```

Comment:
What you can do here is }else{ in 1 line if you really want to follow the coding standard